### DISCUSSION TITLE
Vote for SCORE logo

### DISCUSSION THREAD

**Discussion by antonkri** [eclipse-score/score#47] [Polls]
Option 1:
![image](https://github.com/user-attachments/assets/6cb45960-b9d7-423e-9118-1dd1b7dd635d)

Option 2:
![image](https://github.com/user-attachments/assets/815b2ff1-e515-4b76-954d-a4ea72b614a2)

**Comment by HartmannNico** [comment]
![score 2](https://github.com/user-attachments/assets/2e180ead-fe9c-4ca8-b6ad-894494c54390)
I would opt for 1 and propose to reverse symbol and script

**Reply by pahmann** [reply]
@HartmannNico, I like your proposal the most.

**Reply by antonkri** [reply]
Looks good to me as well

**Comment by antonkri** [comment]
But may be to have it more compact, we could have it like this, @HartmannNico  what do you think:

<img width="555" alt="image" src="https://github.com/user-attachments/assets/8dff852a-094b-4a32-acda-ad9c8c6a5513">

or with some other colours,

<img width="507" alt="image" src="https://github.com/user-attachments/assets/55ae5711-d68a-4777-acb9-86ae1f7e1c9d">

or may be like this:
<img width="642" alt="image" src="https://github.com/user-attachments/assets/4530b1c4-0a3f-4eb2-9d2e-23c03a999e70">

**Comment by qor-lb** [comment]
Hello all,
we would also have some more proposals to be shared:
![image](https://github.com/user-attachments/assets/8f9e1b71-e807-4ab2-9d06-ca79ac0d9065)

A short design explanation from our colleague:
1. Mobility is symbolized by "c" and "o" in the form of wheels.  
2. The visual interplay of C and O represents the flow that emerges when there is a common foundation upon which things can be built and further developed (= Open Source).  
3. The infinity symbol also conveys a sense of unity that connects everyone (a common platform / Open Source).  
   - Variant in the upper line = the "real" infinity symbol, which reduces readability.  
   - Variant in the lower line = a slightly interrupted infinity symbol, which improves readability.  
4. The color gradient is intended to show the dynamism created by offering the software as Open Source (also in line with our latest website colors).  
5. Left half: "score" is written in lowercase -> I find this more harmonious and “rounder” — it also fits the spelled-out project name, which is given in lowercase as well.  
   Right half: "SCORE" is in uppercase -> this is also okay, but in my opinion a bit more cumbersome. That’s just a matter of taste.

**Reply by antonkri** [reply]
Oooh, looks really amazing

**Reply by qor-lb** [reply]
Lower left option with and without a dash:
![SCORE_Logo_gradient](https://github.com/user-attachments/assets/d6f76d97-7522-457f-8fe1-c1f69c163460)

**Reply by qor-lb** [reply]
Some more options:
<img width="992" alt="SCORE_Logo_Varianten" src="https://github.com/user-attachments/assets/8893708a-74ba-4c77-8686-7acd8e35cc09" />
<img width="1913" alt="SCORE-Logo_nur Punkt" src="https://github.com/user-attachments/assets/2d768148-41f6-40ed-9f97-6c087132ac82" />

**Reply by qor-lb** [reply]
The selected option is (see #214)
![image](https://github.com/user-attachments/assets/fe39fdef-990a-4bbd-a297-036a648871d1)

**Comment by FScholPer** [comment]
Can we please stop the voting until all options are in the round and set a duedate until new options can be added
